metis-core/metis-core-common/src/main/java/eu/europeana/metis/core/dataset/PublicationFitness.java

```diff
@@ -0,0 +1,11 @@
+package eu.europeana.metis.core.dataset;
+
+/**
+ * Whether a dataset is fit for publication. {@link #PARTIALLY_FIT} means that some records may be
+ * unfit for publication.
+ */
+public enum PublicationFitness {
```

Comment:
Maybe we should make this a static nested enum inside the dataset class, what do you think?

Reply:
Sure. Why not.